src/containers/HubiiApiHoc/tests/saga.test.js

```diff
@@ -229,6 +229,15 @@ describe('hubiiApi saga', () => {
     beforeEach(() => {
       saga = testSaga(loadWalletBalances, { address }, currentNetworkMock.walletApiEndpoint);
     });
+
+    it('should select currentNetwork inside the saga network if noPoll is true', () => {
```

Comment:
Just for my information, why the network selection depends on this noPoll flag?

Reply:
Usually, it's passed in via the orchestrator (see ln 135). Doesn't really make a difference